### DISCUSSION TITLE
Example in up.macro might be wrong

### DISCUSSION THREAD

**Discussion by robkuz** [unpoly/unpoly#530] [General]
Hi,

i think the example shown in https://unpoly.com/up.macro is wrong.
Probably due to incomplete updates.

I assume that the definition for the macro `smooth-link` (3rd code-segment) would contain the attributes from the first code segment or vice versa

**Comment by adam12** [comment]
I think you're right. Are you interested in submitting a PR?

**Comment by triskweline** [comment]
I fixed the example.
